dotaservice/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.3.7'
+__version__ = '0.3.8'
```

Comment:
Leave the version updates to me in separate commits bc i will push to pypi too

Reply:
ok